src/fastcs/backends/epics/backend.py

```diff
@@ -7,17 +7,23 @@
 
 
 class EpicsBackend(Backend):
-    def __init__(self, controller: Controller, pv_prefix: str = "MY-DEVICE-PREFIX"):
+    def __init__(
+        self,
+        controller: Controller,
+        pv_prefix: str = "MY-DEVICE-PREFIX",
+        options: EpicsIOCOptions | None = None,
+    ):
         super().__init__(controller)
 
         self._pv_prefix = pv_prefix
-        self._ioc = EpicsIOC(pv_prefix, self._mapping)
+        options = options or EpicsIOCOptions()
+        self._ioc = EpicsIOC(pv_prefix, self._mapping, options=options)
```

Comment:
`EpicsIOC` already handles this so we can just pass through `EpicsIOCOptions | None` here.